provision/provision.go

```diff
@@ -451,6 +451,12 @@ type InterAppProvisioner interface {
 	InternalAddresses(ctx context.Context, a App) ([]AppInternalAddress, error)
 }
 
+// CurrentReplicasGetter implements how to get the current (desired)
+// number of replicas (units) of an app.
+type CurrentReplicasGetter interface {
```

Comment:
@nettoclaudio could you rename to CurrentReplicasProvisioner to be like other interfaces ?

Reply:
Sure